jetty-ee8/pom.xml

```diff
@@ -46,9 +50,10 @@
     <javax.mail.glassfish.version>1.4.1.v201005082020</javax.mail.glassfish.version>
     <jetty.servlet.api.version>4.0.6</jetty.servlet.api.version>
     <jsp.impl.version>9.0.52</jsp.impl.version>
-    <modify-sources-plugin.version>1.0.6</modify-sources-plugin.version>
+    <modify-sources-plugin.version>1.0.7-SNAPSHOT</modify-sources-plugin.version>
```

Comment:
this will need a release of this plugin before we merge that

Reply:
@olamy 

You're awesome for taking such rapid action!  🏅

Reply:
release done. just waiting sync to central repo.

Reply:
update done